.eslintrc.js

```diff
@@ -18,25 +18,26 @@ module.exports = {
   ],
   // add your custom rules here
   rules: {
-    'dot-notation':             'off',
-    'generator-star-spacing':   'off',
-    'guard-for-in':             'off',
-    'linebreak-style':          'off',
-    'new-cap':                  'off',
-    'no-empty':                 'off',
-    'no-extra-boolean-cast':    'off',
-    'no-new':                   'off',
-    'no-plusplus':              'off',
-    'no-useless-escape':        'off',
-    'nuxt/no-cjs-in-config':    'off',
-    'semi-spacing':             'off',
-    'space-in-parens':          'off',
-    strict:                     'off',
-    'unicorn/no-new-buffer':    'off',
-    'vue/html-self-closing':    'off',
-    'vue/no-unused-components': 'warn',
-    'vue/no-v-html':            'off',
-    'wrap-iife':                'off',
+    'dot-notation':                   'off',
+    'generator-star-spacing':         'off',
+    'guard-for-in':                   'off',
+    'linebreak-style':                'off',
+    'new-cap':                        'off',
+    'no-empty':                       'off',
+    'no-extra-boolean-cast':          'off',
+    'no-new':                         'off',
+    'no-plusplus':                    'off',
+    'no-useless-escape':              'off',
+    'nuxt/no-cjs-in-config':          'off',
+    'semi-spacing':                   'off',
+    'space-in-parens':                'off',
+    strict:                           'off',
+    'unicorn/no-new-buffer':          'off',
+    'vue/html-self-closing':          'off',
+    'vue/no-unused-components':       'warn',
+    'vue/no-v-html':                  'off',
+    'vue/multi-word-component-names': 'off',
```

Comment:
Please make our changes after line 160, in the section about Desktop changes; this part is straight from dashboard.

It might be good to update the bits we copied from dashboard too, but that's optional.

Reply:
done